scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java

```diff
@@ -176,6 +188,13 @@ public void schedulerStatsUpdated(HashMap<String, String> stats) {
         r.setAttribute("TotalJobsCount", stats.get("TotalJobsCount"));
         r.setAttribute("PendingJobsCount", stats.get("PendingJobsCount"));
 
+        r.setAttribute("StalledJobsCount", stats.get("StalledJobsCount"));
+        r.setAttribute("PausedJobsCount", stats.get("PausedJobsCount"));
+        r.setAttribute("InErrorJobsCount", stats.get("InErrorJobsCount"));
+        r.setAttribute("KilledJobsCount", stats.get("KilledJobsCount"));
+        r.setAttribute("CancelledJobsCount", stats.get("CancelledJobsCount"));
+        r.setAttribute("FailedJobsCount", stats.get("FailedJobsCount"));
+
```

Comment:
Minor comment: the same remark applies here

Reply:
good